modules/core/src/test/java/org/apache/ignite/internal/metric/SystemViewSelfTest.java

```diff
@@ -289,6 +289,7 @@ public void testServices() throws Exception {
                 assertEquals(TestNodeFilter.class, sview.nodeFilter());
                 assertFalse(sview.staticallyConfigured());
                 assertEquals(g.localNode().id(), sview.originNodeId());
+                assertEquals(F.asMap(g.localNode().id(), 1), sview.topologySnapshot());
```

Comment:
Btw. Can we also remove repeated 'assertEquals(srvcCfg.getMaxPerNodeCount(), sview.maxPerNodeCount());' above ?

Reply:
Done